lib/mrsk/sshkit_with_ext.rb

```diff
@@ -1,9 +1,19 @@
 require "sshkit"
 require "sshkit/dsl"
 
+module AppHelper
+  def stale_versions(role:)
```

Comment:
Let's use the `cli = self` pattern that you can find a few other places in the codebase instead 👍

Reply:
We could also make it private but the issue is that at that point I'm not sure on how to leverage the on(host) parallelism.

Reply:
I used the `no_commands` block to make it explicit that it's not a command